shared/common-adapters/header-hoc.native.js

```diff
@@ -10,10 +10,17 @@ import type {Props} from './header-hoc'
 
 function HeaderHoc<P: {}>(WrappedComponent: React.ComponentType<P>) {
   const HeaderHocWrapper = (props: P & Props) => {
-    const {onBack, onCancel, headerStyle, title, customComponent, theme = 'light'} = props
+    const {onBack, onCancel, headerStyle, title, customComponent, theme = 'light', showBorder = true} = props
     return (
       <Box style={_containerStyle}>
-        <Box style={{..._headerStyle, ..._headerStyleThemed[theme], ...headerStyle}}>
+        <Box
+          style={{
+            ..._headerStyle,
+            ..._headerStyleThemed[theme],
+            ...headerStyle,
+            borderBottomWidth: showBorder ? StyleSheet.hairlineWidth : 0,
```

Comment:
this might not be safe as other components likely did this with the headerStyle prop instead, lets discuss

Reply:
oh I completely missed that L20 didn't have an underscore, that makes more sense